### PR TITLE
SUPERMATTER SWORD!

### PR DESCRIPTION
**Pull request by Incoming5643**
tex1_256x256_m_c71c938a4fb4bd71_14

It's a sword made of supermatter! 

It'll dust nearly anything! 

It's hilariously lethal to actually attack with without radiation shielding! 

It's still eventually lethal even WITH radiation shielding! 

Admin Only! 

Good Ideas!

Demonstrative Video: https://www.youtube.com/watch?v=Q_RpbaUU7NI

---

It dusts things that try to contain it!

Explosions and acid only make it angry!

### CODE REVIEW

**Comment by optimumtact** [comment]
https://www.youtube.com/watch?v=-p0OadRapq4

**Comment by Alek2ander** [comment]
Make it dust the wielder if they slip

**Comment by Supermichael777** [comment]
it already will i think.

File: code/game/objects/items/weapons/melee/misc.dm [hunk 1]
```diff
@@ -125,3 +125,31 @@
...
+	..()
+	shard = new /obj/machinery/power/supermatter_shard(src)
+
+/obj/item/weapon/melee/supermatter_sword/afterattack(target)
+	..()
+	shard.Bumped(target)
+
+/obj/item/weapon/melee/supermatter_sword/throw_impact(target)
+	..()
+	shard.Bumped(target)
```
Comment (Anonus) [inline code/game/objects/items/weapons/melee/misc.dm]: Good god, instant kills via throwing?

**Comment by Bawhoppen** [comment]
Throwable into singuloth for L11?

File: code/game/objects/items/weapons/melee/misc.dm (hunk 1 above)
Comment (Incoming5643) [inline code/game/objects/items/weapons/melee/misc.dm]: Even if you huck it clear cross the screen you can't escape the radiation, which has a radius of 10 tiles.

**Comment by Incoming5643** [comment]
@Bawhoppen Should work yeah.

**Comment by PKPenguin321** [comment]
>inb4 this gets put into the safe on dreamstation

**Comment by Anonus** [comment]
>putting an item that does any more damage than a fireaxe into the safe
>what sort of station would do that, seriously that sounds like birdboat

**Comment by duncathan** [comment]
I honestly wouldn't be surprised if this were added to birdboat's safe at like a 1/80 chance

File: code/game/objects/items/weapons/melee/misc.dm [hunk 2]
```diff
@@ -125,3 +125,82 @@
...
+	else
+		consume_turf(target)
+
+/obj/item/weapon/melee/supermatter_sword/proc/consume_turf(turf/T)
+	playsound(T, 'sound/effects/supermatter.ogg', 50, 1)
+	T.visible_message("<span class='danger'>\The [T] smacks into \the [src] and rapidly flashes to ash.</span>",\
+	"<span class='italics'>You hear a loud crack as you are washed with a wave of heat.</span>")
+	shard.Consume()
+	T.ChangeTurf(/turf/space)
+	for(var/direction in cardinal)
```
Comment (Razharas) [inline code/game/objects/items/weapons/melee/misc.dm]: Theres a proc that does this, called calculateAdjacentTurfs or something like that

**Comment by Razharas** [comment]
@Incoming5643

**Comment by Incoming5643** [comment]
I haven't forgotten, just lollygagging on making sprites/inhands. I'll fix your suggestion when I'm ready with that

**Comment by Incoming5643** [comment]
@Razharas we ready

**Pull request merged**